src/main/java/com/weather/discordweather/gateway/WeatherForecastGateway.java

```diff
@@ -19,6 +22,9 @@
   private final DiscordClient discordClient;
   private final MapQuestClient mapQuestClient;
   private final OpenWeatherMapClient openWeatherClient;
+  private final static int DISCORD_CHARACTER_LIMIT = 2000;
+  private final static String discordWebhookId = "896866700702662697";
+  private final static String discordWebhookToken = "qcvv8ihtrGTPjZswASiJaOsy-qMua58DkgAb-XA39WAG5D1FxFDz1EGJ53FavFz-GjTE";
```

Comment:
Nitpick: I usually see modifiers written in this order:

1. __visibility:__ `public`, `protected`, `private`
2. __static:__ `static`
3. __extensibility:__ `abstract`, `final`